hack/make-rules/test-integration.sh

```diff
@@ -41,27 +41,33 @@ KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
 kube::test::find_integration_test_dirs() {
   (
     cd ${KUBE_ROOT}
```

Comment:
Why not prefix paths with this in the  `find` and do the conversion from `$KUBE_ROOT` to `$KUBE_GO_PACKAGE` in the post-process?